x-pack/plugins/cases/server/services/user_actions/operations/create.ts

```diff
@@ -399,15 +395,56 @@ export class UserActionPersister {
         payload,
       });
 
-      if (userAction) {
-        await this.createAndLog({ userAction, refresh });
+      if (userActionPayload) {
+        await this.createAndLog({ userAction: userActionPayload, refresh });
       }
     } catch (error) {
       this.context.log.error(`Error on creating user action of type: ${type}. Error: ${error}`);
       throw error;
     }
   }
 
+  public async bulkCreateUserAction<T extends keyof BuilderParameters>({
```

Comment:
Could you add some tests for this function? (`x-pack/plugins/cases/server/services/user_actions/operations/create.test.ts`)